docs/source/conf.py

```diff
@@ -19,7 +19,7 @@ class UnsetType:
 
 
 project = "msgspec"
-copyright = "2022, Jim Crist-Harif"
+copyright = "%Y, Jim Crist-Harif"
```

Comment:
You don't need a copyright year, and if anythiing it should be the year first published, not the current year. Better off to just remove the year entirely.

```suggestion
copyright = "Jim Crist-Harif"
```